File: android-app/src/main/java/fr/cph/chicago/entity/dto/TrainArrivalDTO.java
package fr.cph.chicago.entity.dto;

import android.util.SparseArray;
import fr.cph.chicago.entity.TrainArrival;
import lombok.Builder;
import lombok.Data;

/**
 * @author cpharmant
 */
@Builder
@Data
public class TrainArrivalDTO {
    private boolean error;
    private SparseArray<TrainArrival> trainArrivalSparseArray;
}

File: android-app/src/main/java/fr/cph/chicago/entity/dto/TrainArrivalDTO.kt
package fr.cph.chicago.entity.dto

import android.util.SparseArray
import fr.cph.chicago.entity.TrainArrival

/**
 * @author cpharmant
 */
class TrainArrivalDTO(val trainArrivalSparseArray: SparseArray<TrainArrival>? = null, val error: Boolean = false)
